frontend_server/lib/server.dart

```diff
@@ -284,27 +281,29 @@ Future<int> starter(List<String> args, {
   _State state = _State.READY_FOR_INSTRUCTION;
   String boundaryKey;
   input
-    .transform(UTF8.decoder)
-    .transform(new LineSplitter())
-    .listen((String string) async {
+      .transform(UTF8.decoder)
+      .transform(new LineSplitter())
+      .listen((String string) async {
     switch (state) {
```

Comment:
This is another example. The indenting suggests that the `switch` is at a higher level than the `.listen` call, but it's actually the other way around (`switch` is inside the `listen` call).

Reply:
I've added this one to our list too.